framework/source/class/qx/core/Property.js

```diff
@@ -991,7 +1036,7 @@ qx.Bootstrap.define("qx.core.Property",
       );
 
       if (incomingValue) {
-        code.push('if(this.', store, '===value)return value;');
+        code.push('if(equ(this.', store, ',value))return value;');
       } else if (resetValue) {
```

Comment:
Ah! I see! and **yes** it **should** be called with the instances context!
Thanks for the hint